torch/ao/quantization/observer.py

```diff
@@ -1183,6 +1183,11 @@ def forward(self, x_orig: torch.Tensor) -> torch.Tensor:
             assert (
                 combined_min.numel() == 1 and combined_max.numel() == 1
             ), "histogram min/max values must be scalar."
+
+            # TODO: For some reason, this is required for it to pass torchscript test
```

Comment:
thanks, the current code didn't do detach() actually, maybe you can add the detach() back as well